addons/google_calendar/utils/google_calendar.py

```diff
@@ -71,14 +72,27 @@ def get_events(self, sync_token=None, token=None, event_id=None, timeout=TIMEOUT
 
         return GoogleEvent(events), next_sync_token, default_reminders
 
+    def override_record_with_values(self, request_values):
+        env = request.env
+        event_values = request_values[1]
+        google_meet_link = event_values.get('hangoutLink')
+        odoo_record_id = event_values.get('extendedProperties', {}).get('shared', {}).get('%s_odoo_id' % env.cr.dbname)
```

Comment:
I think you can use `gevent = GoogleEvent(request_values)`  to get an object and easily get the odoo record associated. You will have less custom code or copy paste.  What do you think ?